src/bindings/jna/README.md

```diff
@@ -174,14 +174,18 @@ You can run unit tests by invoking:
 ./gradlew test
 ```
 
-## Limitations
-
-- no Java iterator for metadata
-
 ## Release publishing to Maven Central
 
 Release publishing is done via `buildAndPublishMaven()` in [Jenkinsfile.release](../../../scripts/jenkins/Jenkinsfile.release).
 
+## Upgrading Gradle
+
+To upgrade the Gradle version used by the JNA bindings, the following steps should be considered:
+
+- [Update the gradle wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:upgrading_wrapper) for the Java binding
+- Search all [Dockerfiles](../../../scripts/docker) and replace the Gradle version there
```

Comment:
I've always used something like
```
grep -rl --exclude-dir="build" "gradle-7.1.1" | xargs sed -i 's/gradle-7.1.1/gradle-7.2/g'
```
and then reviewed the changes before committing.

Maybe we could add a script so that this is easier than looking into all files manually. (not for this PR)

Reply:
I would just test and your proposed command and include it in the README.md with #3988 

would that be ok?

Reply:
@tucek I think this is a great idea! :sparkling_heart: